tests/console/python3_beautifulsoup4.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: python3-beautifulsoup tests
+# - install python-beautifulsoup package
+# - use library to parse sample htmlfile
+# - Compare the result vs expected
+#
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'zypper_call';
+
+sub run {
+    my $self = shift;
+    select_serial_terminal;
+    # Make sure beautifulsoup python module is installed
+    zypper_call "in python3 python3-beautifulsoup4 python3-lxml";
```

Comment:
Can you please separate the verification system's default Python version from the available Python versions? Only install the available Python version.